### DISCUSSION TITLE
Noah-3.6 with HYMAP routing stops

### DISCUSSION THREAD

**Discussion by sclee1717** [NASA-LIS/LISF#857] [Q&A] [answered]
Hi all,

I'm trying to run LIS, with Noah-3.6 and HYMAP router, and getting the following statements.

----------------------------------------------------------------------------------------------------------------------
.... 
 [INFO] GDAS -- changing grid to 2015 (T1534) --
 MSG: The GDAS forcing resolution is finer than the running domain.
      Upscaling with the average method.
 [INFO] Getting new time1 data
 [INFO] Reading GDAS file1 (I) /data/lisdata/METFORCING/gdas/201601/2016010100.gdas1.sfluxgrbf00.sg
 [INFO] Reading GDAS file1 (A) /data/lisdata/METFORCING/gdas/201512/2015123118.gdas1.sfluxgrbf03.sg
 [INFO] Reading GDAS file1 (A)/data/lisdata/METFORCING/gdas/201512/2015123118.gdas1.sfluxgrbf06.sg
 [INFO] Getting new time2 data
 [INFO] Reading GDAS file2 (I) /data/lisdata/METFORCING/gdas/201601/2016010100.gdas1.sfluxgrbf03.sg
 [INFO] Reading GDAS file2 (A) /data/lisdata/METFORCING/gdas/201601/2016010100.gdas1.sfluxgrbf03.sg
 [ERR] m3 s-1 for field streamflow is not defined 
 [ERR] for diagnostic output...
 [ERR] supported unit types: n3 s-1              
 [ERR] Program stopping ..
 [ERR] endrun is being called
----------------------------------------------------------------------------------------------------------------------

[files.zip](https://github.com/NASA-LIS/LISF/files/6876296/files.zip)

Any suggestions regarding how to solve or what could I check for, would be greatful.
I'm attaching the files used for my LDT and LIS runs.

Thanks a lot.
Lee

**Comment by agetiran** [comment, marked as answer]
Hi,
the issue is in your output list table, it seems. When you define the HyMAP output variables, please use the line below instead:

Streamflow:      1 m3/s    -     1 0 0 1 333 10       #Streamflow

**Reply by sclee1717** [reply]
Hi!
Thanks for your answer. However, the output list table already is written:
Streamflow:   1  m3/s    -    1 0 0 1 333 10      # Streamflow

btw, I just have found that there's no problem using HYMAP2 router, with everything else being the same.
still am trying to figure out the reason of the error using HYMAP router,,

THanks a lot.
Lee

**Reply by agetiran** [reply]
The use of HyMAP has been discouraged as it is an old version with limited capabilities. Instead, I would recommend that you use HyMAP2 as it is a more advanced model. Let me know if you have other questions.

Augusto

**Reply by sclee1717** [reply]
Thank you!

Actually there is one more thing I'm curious of.
When I was trying various options using HYMAP router, I faced number of errors that starts with:
1. Operating system error: cannot allocate memory
2. set sws routine for LSM + routing instance Noah.3.6+HYMAP router is not defined; program will seg fault....

I was guessing there's something mistaken in the configuration? but there were no such problems using HYMAP2.
Are there any chances that the errors arose also because HYMAP is an old version?

Lee

**Reply by agetiran** [reply]
That's correct. Error 2 is related to the 2-way coupling, only available for HyMAP2.
